api/institutions/views.py

```diff
@@ -474,7 +474,8 @@ class InstitutionDepartmentList(InstitutionImpactList):
     serializer_class = InstitutionDepartmentMetricsSerializer
     renderer_classes = tuple(api_settings.DEFAULT_RENDERER_CLASSES, ) + (InstitutionDepartmentMetricsCSVRenderer, )
 
-    ordering = ('-number_of_users', 'name',)
+    ordering_fields = ('-number_of_users', 'name', )
```

Comment:
shouldn't need the `-` here, but also looks like it doesn't hurt

Reply:
Agreed, we can change this later, otherwise we need to wait for another round of CI.

Reply:
hmm, got a 401 when making `http://localhost:8000/v2/institutions/cos/metrics/departments/`requestion after removing the `-`, so must have been there for some reason.